app/models/in_person_enrollment.rb

```diff
@@ -22,6 +22,25 @@ class InPersonEnrollment < ApplicationRecord
   before_save(:on_status_updated, if: :will_save_change_to_status?)
   before_create(:set_unique_id, unless: :unique_id)
 
+  def self.pending_established(early_benchmark, late_benchmark)
+    where(status: :pending).
+      and(
+        where.not(enrollment_established_at: nil).
```

Comment:
I think you might not need to check that `enrollment_established_at` is nil before you check it for a value?

Reply:
i think Tim suggested adding it as an extra check